pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -169,74 +185,124 @@ func (r *ReconcileBackup) createVeleroBackupObject(namespace string) error {
 			IncludedNamespaces: []string{
 				namespace,
 			},
-			ExcludedResources: DefaultExcludedBackupResources,
+			ExcludedResources: defaultExcludedBackupResources,
 		},
 	}
 
 	return r.Create(context.TODO(), backup)
 }
 
-func (r *ReconcileBackup) getModifiedHiveObjectsInNamespace(namespace string) ([]*hiveObject, error) {
-	nsLogger := addNamespaceLoggerFields(r.logger, namespace)
-
-	filteredHiveObjects := []*hiveObject{}
+func addNamespaceLoggerFields(logger log.FieldLogger, namespace string) log.FieldLogger {
+	return logger.WithField("namespace", namespace)
```

Comment:
This field should be included in the logger for all of the logs in `Reconcile`. It should be added at the same time as the fields in `addReconcileRequestLoggerFields` are added. At that point, there is no need for either this function or `addReconcileRequestLoggerFields` as they would each only be called once and the replacement code inline would be just as concise.

Reply:
@staebler Should the logger with the namespace field be passed from Reconcile down into each function it calls?

Or should r.logger be updated to include the field?

Not sure if r.logger should be updated as I believe that's an object that is shared between all calls to reconcile, which can be run simultaneously.

Reply:
Definitely don't want to mutate `r.logger`. Whether you pass the logger down as a parameter or set it up in each function is your choice. The former aligns with how it is done in other controllers. The latter makes calling signatures cleaner, until there are fields that you want to include that are not already available in the called function.

Reply:
Looking more closely at the logging, I see now that the logger used in `Reconcile` does indirectly include the namespace in the "request" field. It would be better, however, if the namespace were consistent throughout all of the logs generated by this controller. So either all of the logs use the "request" field or all of the logs use the "namespace" field.

There are also logs like [here](https://github.com/openshift/hive/blob/2913069b622b7d9fbc27db14d34e5b0dbdfe5395/pkg/controller/velerobackup/velerobackup_controller.go#L282) that do not include either field. It should include whichever you choose as well.